.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,59 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels: 'type: bug'
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## First of all
+        1. Please search for [open issues](https://www.github.com/tauri-apps/tauri/issues) about this problem first.
+        2. Make sure `rustc` and all relevant packages are up to date.
```

Comment:
```suggestion
        2. Make sure `rustc` and all relevant Tauri packages are up to date.
```

Reply:
Consciously removed it to make it shorter and more easily reusable. Think it's necessary?

Reply:
I think so and it is not that long anyway.